go.mod

```diff
@@ -48,6 +48,6 @@ replace (
 	github.com/earthly/earthly/ast => ./ast
 	github.com/jessevdk/go-flags => github.com/alexcb/go-flags v0.0.0-20210722203016-f11d7ecb5ee5
 
-	github.com/moby/buildkit => github.com/earthly/buildkit v0.0.1-0.20220913002628-9c256c19c539
+	github.com/moby/buildkit => github.com/earthly/buildkit v0.0.1-0.20220922172718-e3950730ae66
```

Comment:
why is buildkit being updated here?

Reply:
Oh I don't even know where that came from. Glad you noticed.